Scala/Scala.sublime-syntax

```diff
@@ -138,7 +138,7 @@ contexts:
           pop: true
         - match: '([a-zA-Z$_][a-zA-Z0-9$_]*)\s*:\s*'
           captures:
-            1: entity.name.variable variable.parameter
+            1: entity.name.parameter variable.parameter
```

Comment:
> Indexing every parameter in a codebase feels wrong, but then again, I don't know Scala at all.

It is a little noisy, in practice, but not so noisy that I feel it's necessarily wrong.  Are function parameters indexed in the Java, Ruby, Python or Haskell modes?